src/redux/reducers/settings.js

```diff
@@ -16,19 +15,31 @@ const defaultState = {
   userName: '',
   prefLanguage: 'en-US',
   timeZone: 'UTC-02',
-  themeValues: '',
+  customThemeValue: [
```

Comment:
Could we use this instead of this array? 
```
{
    header: '#4285f4',
    pane: '#f3f2f4',
    body: '#fff',
    composer: '#f3f2f4',
    textarea: '#fff',
    button: '#4285f4',
    textColor: '#000',
  }
```
As it seems the other components such as MessageSection is using these conventions, plus it will be really tough to decide which color is for what component if you save them like this.

Reply:
Okay, will make the appropriate changes